xmltojsonlegacy/src/main/java/org/assimbly/xmltojsonlegacy/CustomXmlJsonDataFormat.java

```diff
@@ -21,13 +21,27 @@
 import java.io.OutputStream;
 import java.io.StringReader;
 import java.io.StringWriter;
+import java.util.HashMap;
+import java.util.Set;
 
 public class CustomXmlJsonDataFormat implements DataFormat {
 
     static final Logger logger = Logger.getLogger(CustomXmlJsonDataFormat.class);
 
     // XML to JSON options
     private boolean forceTopLevelObject, skipWhitespace, trimSpaces, skipNamespaces, removeNamespacePrefixes, typeHints;
+    // special case
+    private boolean isSpecialCaseTTFTTF = false;
+
+    // set with options to discard
+    private Set<String> discardXmlToJsonOptionsSet = Set.of(
```

Comment:
Don't really understand to put all options in to Set<String>. Why not used boolean getters/setters?

Reply:
Maybe don't only use a processor, but really turn it into a component (with a component class, endpoint class en configuration class).

For example: https://github.com/assimbly/custom-components/tree/develop/xmltocsv/src/main/java/org/assimbly/xmltocsv

Reply:
> Don't really understand to put all options in to Set. Why not used boolean getters/setters?

yes, I can use the get methods to retrieve the values and use some if logic cases, but I think It's more readable like this.